cpp/modmesh/CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ add_subdirectory(python)
 add_subdirectory(spacetime)
 add_subdirectory(view)
 add_subdirectory(inout)
+add_subdirectory(testhelper)
```

Comment:
some tests require python binding functions or classes. We can put those test helpers in this `testhelper` module in the future.